dist/tests/if_lib/philip_if.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2018 Kevin Weiss <kevin.weiss@haw-hamburg.de>
+# Copyright (c) 2018 Kevin Weiss, for HAW Hamburg  <kevin.weiss@haw-hamburg.de>
```

Comment:
Usually we add just HAW and put the author info into the @author tag. The copyright holder is either Kevin or HAW, not "Kevin for HAW".

Reply:
If Kevin did part of the work in work hours and part at home, adding both would make most sense.